validate.py

```diff
@@ -136,6 +136,11 @@ def determine_if_winner():
 
   scalar = min(0.00001, 1.0 / total_points)
   points_so_far = 0
+  
+  print('Probability of winning:')
+  for user, user_points in summed_user_points:
+    print('%s: %.3f%%' % (user, user_points * scalar * 100))
```

Comment:
nit: this would be slightly better as `print('  %s:` because then instead of:

```
Probability of winning:
jburger424: 0.011%
jeffkaufman: 0.018%
pavellishin: 0.012%
tnelling: 0.011%
```

you would get:

```
Probability of winning:
  jburger424: 0.011%
  jeffkaufman: 0.018%
  pavellishin: 0.012%
  tnelling: 0.011%
```